components/colorwheel/index.css

```
@@ -56,35 +94,68 @@
   }
 }
 
-.spectrum-ColorWheel-gradient {
-  position: relative;
-  z-index: 0;
-  width: var(--spectrum-colorwheel-width);
-  height: var(--spectrum-colorwheel-height);
-  border-radius: 100%;
-  box-sizing: border-box;
-
-  border-width: var(--spectrum-colorwheel-border-size);
-  border-style: solid;
+.spectrum-ColorWheel-internal{
+  display: flex;
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
+  width: var(--spectrum-colorwheel-colorarea-container-size);
+  height: var(--spectrum-colorwheel-colorarea-container-size);
+  margin: auto;
 }
 
-.spectrum-ColorWheel-wheel {
-  position: relative;
-  z-index: 1;
-}
-
-.spectrum-ColorWheel-outerCircle,
-.spectrum-ColorWheel-innerCircle {
-  fill: transparent;
-
-  stroke-width: var(--spectrum-colorwheel-border-size);
+.spectrum-ColorWheel-colorarea-container{
+  height: auto;
+  width: 100%;
+  margin: var(--mod-colorwheel-colorarea-margin, var(--spectrum-colorwheel-colorarea-margin));
 }
 
 .spectrum-ColorWheel-slider {
   @extend %spectrum-ColorControl-hiddenField;
 }
 
 .spectrum-ColorWheel-handle {
+  transform: translate(var(--spectrum-colorwheel-colorhandle-position), 0);
```

Can you also add the `--mod-` prefixed custom property reference here?

```suggestion
  transform: translate(var(--mod-colorwheel-colorhandle-position, var(--spectrum-colorwheel-colorhandle-position)), 0);
```